sdk/servicebus/service-bus/test/internal/atomXml.spec.ts

```diff
@@ -1088,7 +1088,7 @@ describe(`Parse empty response for list() requests to return as empty array`, fu
         headers: new HttpHeaders({})
       };
     };
-    const result = await mockServiceBusAtomManagementClient.getQueues();
+    const result = await mockServiceBusAtomManagementClient["listQueues"]();
```

Comment:
Why do we need to call into the private API for this?

Reply:
This was the older getQueues() API that takes skip and top as input params which is hidden now and is privately used by the newer ones that return PageAsyncIterableIterator.

These tests didn't seem to make sense for the iterator APIs, so I just retained these tests by calling into the private API.

Reply:
Do we need these tests anymore then? Would it be better to just remove them?

Reply:
The methods still exist though private and are being used by the new listing methods, so I'd like to keep them as is.